emcc/pre.js

```diff
@@ -0,0 +1,6 @@
+var Module = {
+    'ready': false,
+    'print': function(text) { console.log('jsdrafter.js: ' + text); },
+    'printErr': function(text) { console.error('jsdrafter.js: ' + text); },
```

Comment:
Might consider renaming these to `log` and `logErr`. Presumably when you override them you want to log information and not necessarily print it to the screen.